src/framework/statusDropdown/StatusDropdown.jsx

```diff
@@ -31,7 +31,7 @@ export default class StatusDropdown extends Component {
       [StatusDropdown.OPTIONS.ACTIVATE]: StatusDropdownOptionIcon.TYPE.ACTIVATE,
```

Comment:
I'd fill out the other maps alright grey is the default color but it makes it way clearer.